tests/tests.py

```diff
@@ -230,6 +229,7 @@ def test_single_form_csv_with_rep_fields_start_with_repeat(self):
         for line1, line2 in izip(open(get_filename('models.py'),'r'),open(get_filename(cmp_fileName),'r')):
             self.assertEqual(line1,line2)
     def test_single_form_csv_with_rep_fields_many_nested(self):
```

Comment:
Ok. I would remove the triple quotes (since that is interpreted as a docstring) and comment out the whole method. Add a comment above it stating this fact.